src/hydra/addParameters.js

```diff
@@ -0,0 +1,40 @@
+import Parameter from "../Parameter";
+import fetchResource from "./fetchResource";
+
+export default api => {
+  const promises = [];
+
+  for (const resource of api.resources) {
+    let promise = fetchResource(resource.url).then(response => {
+      if (!response.parameters) {
+        return [];
+      }
+
+      const resourceParameters = [];
+
+      response.parameters.forEach(parameter => {
+        const property = parameter.property;
+
+        if (property === null) {
+          return;
+        }
+
+        resourceParameters.push(
+          new Parameter(parameter.variable, "", parameter.required, "")
```

Comment:
@dunglas : what is the `type` value you are expecting ?

Reply:
Something like an XSD type?

Reply:
It will be the same that the `range` attribute of the `Field`

Reply:
Tests fixed & and attribute name changed to `range`